src/blocks/Image/ZestyImage.js

```diff
@@ -14,20 +15,30 @@ import FillerContent from 'components/globals/FillerContent';
 
 const ZestyImage = ({ options, src, alt, loading, style, attributes }) => {
   /* Taking the options object and converting it into a query string. */
-  const imageUrl = Object.entries(options).reduce(
-    (acc, item, idx) => {
-      const newUrl = `${acc}${idx === 0 ? '?' : '&'}${item[0]}=${item[1]}`;
-      return newUrl;
-    },
-    src ? src : FillerContent.photos[0].src,
-  );
+  const { width, height } = { ...options };
+  const [url, setUrl] = useState('');
+
+  // ignore if options params is empty or not defined
+  if (options) {
+    useMemo(() => {
+      const imageUrl = Object.entries(options).reduce(
+        (acc, item, idx) => {
+          const newUrl = `${acc}${idx === 0 ? '?' : '&'}${item[0]}=${item[1]}`;
+          return newUrl;
+        },
+        src ? src : FillerContent.photos[0].src,
+      );
+
+      setUrl(imageUrl);
+    }, [options]);
+  }
 
   return (
     <img
-      width={options.width}
-      height={options.height}
+      width={width}
+      height={height}
       {...attributes}
-      src={imageUrl}
+      src={url ? url : src}
       alt={alt}
       loading={loading}
       style={style}
```

Comment:
@ardeay I've destructured the width and height from options object if exist if not it will just work as normal img tag, the code earlier if you did not pass the options object it will throw an error from object entries method because it's empty now we can possibly use the component without options object.